### ISSUE TITLE
typo in bash cheat sheet

### ISSUE THREAD

**Issue by mrichardson3** [remigiusz-suwalski/programming-cheatsheets#1]
on line starting with "less reads files", "view" is spelled "vew".  Thanks for the cheat sheet, too!  I learned some stuff I didn't know and remembered some arcane stuff that had slipped away from me.

**Comment by remigiusz-suwalski** [comment]
The commit #64b5bddb0e4099d1a739f1e420d018bad3d47477 fixes this typo and explains, why less should be used instead of more.

**Comment by mrichardson3** [comment]
Dear Remigiusz,

I also thought of one other thing that possibly should be added. I think 
the "locate" command uses an internal database that normally gets 
updated via a daily cron job.  However, if you've recently added a bunch 
of files and want to force an update, the command to use is "updatedb".  
I'm not sure if this command meets your rubric for inclusion in your 
wonderful cheat sheet. Thanks again!

over and out,
Mark Richardson.

**Comment by remigiusz-suwalski** [comment]
Of course this command meets my requirements. By accident, the commit 852547e55a11e6dd0e3e550e9aaaaf361a493d5e merges changes proposed by ellie195 into main branch and therefore your request has been satisfied.

Is the description sufficient? Are any of important flags for `updatedb` missing?

**Comment by mrichardson3** [comment]
I looked to see if there were any important flags for updatedb. Although 
there are many flags, I don't find any of them worth mentioning. I did 
fall down the rabbit hole a bit as it appears the "locate" command 
really calls the "mlocate" command (at least with Ubuntu for the past 15 
years). For giggles, I went to 
unix.stackexchange.com/questions/273182/difference-between-locate-and-mlocate 
to try and figure the differences between the two commands. Ultimately, 
I think the differences are not significant for the average user and the 
cheat sheet is fine as it is.

over and out,
Mark
